### ISSUE TITLE
Crash on start: The driver type 'USBPRINT' is not supported yet

### ISSUE THREAD

**Issue by nichu42** [Nitrokey/nitrokey-app2#288]
I'm getting the following error since upgrading to the latest version:

```
Failed to execute script '_main_' due to unhandled exception: The driver type 'USBPRINT' is not supported yet

Traceback (most recent call last):
  File "nitrokeyapp\__main__.py", line 40, in <module>
  File "nitrokeyapp\__main__.py", line 34, in main
  File "nitrokeyapp\gui.py", line 42, in __init__
  File "usbmonitor\usbmonitor.py", line 35, in __init__
  File "usbmonitor\__platform_specific_detectors\_windows_usb_detector.py", line 30, in __init__
  File "usbmonitor\__platform_specific_detectors\_usb_detector_base.py", line 43, in __init__
  File "usbmonitor\__platform_specific_detectors\_windows_usb_detector.py", line 46, in get_available_devices
  File "usbmonitor\__platform_specific_detectors\_windows_usb_detector.py", line 83, in __finetune_incompatible_attributes
  File "usbmonitor\__platform_specific_detectors\_windows_usb_detector.py", line 123, in __get_driver_type_from_device_id
AssertionError: The driver type 'USBPRINT' is not supported yet, please create an issue in github
```

**Comment by daringer** [comment]
Oh wow, looks like the `usbmonitor` package doesn't like a specific usb-connected-device ?

Do you by any chance have a usb printer connected to your computer? 
Could you try disconnecting it and restart nitrokey-app2 ?
It also makes sense that this occurs starting from this version as we added `usbmonitor` to have a consistent device detection across all OSes.

**Comment by nichu42** [comment]
Yes, I have two Dymo label printers connected via USB. I can start Nitrokey App if they are disconnected upon start of the app.
Installation was through downloaded MSI file.

Happy to send you more data. Just let me know what to do.

**Comment by serial** [comment]
Hi,
I have exactly the same problem on Windows with versions 2.3.3 no matter if installer or portable. A Dymo label printer is also connected here. Deactivating in the device manager is not enough, the device must be completely disconnected.

Is there anything new on this yet?

**Comment by robin-nitrokey** [comment]
Looks like this issue has been fixed upstream in usb-monitor v1.23.  I’ve created a PR to update the dependency: https://github.com/Nitrokey/nitrokey-app2/pull/318.

**Comment by mmerklinger** [comment]
Fixed in version 2.3.5.
